hr_request_overtime/models/hr_overtime.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models, api, fields, _
+from odoo.exceptions import UserError
+
+
+class HROvertime(models.Model):
+    _name = 'hr.overtime'
+
+    name = fields.Char(string="Name")
+    employee_id = fields.Many2one('hr.employee', string="Employee")
+    project_id = fields.Many2one('project.project', string="Project")
+    time_report_id = fields.Many2one('hr_timesheet.sheet', string="Time Report")
+    hours = fields.Float(string="Hours")
+    type = fields.Selection([('Wanted', 'Wanted'), ('Ordered', 'Ordered')], string="Type", default='Wanted')
+    state = fields.Selection([('draft', 'Draft'), ('submit', 'To Approve'),
```

Comment:
I think it should be submitted?
submit -> submitted

Reply:
updated